addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -74,10 +74,20 @@
         <td align="center" style="min-width: 590px;">
             <table border="0" cellpadding="0" cellspacing="0" width="100%" style="background-color: white; padding: 0; border-collapse:separate;">
                 <tr><td valign="middle">
-                    <span style="font-size: 10px;">Your <t t-esc="model_description or 'document'"/></span><br/>
-                    <span style="font-size: 20px; font-weight: bold;">
-                        <t t-esc="message.record_name and message.record_name.replace('/','-') or ''"/>
-                    </span>
+                    <span style="font-size: 10px;">Your <t t-esc="model_description or 'document'"/></span>
+                    <br/>
+                    <t t-if="has_button_access">
+                        <a t-att-href="button_access['url']">
+                            <span style="font-size: 20px; font-weight: bold;">
+                                <t t-esc="message.record_name and message.record_name.replace('/','-') or ''"/>
```

Comment:
I see that the other template supports:

`button_access['title']`

Here we force the record name?
I'm not an expert on those so it may be perfectly normal.

Reply:
Good question! If I remember well, button title contains "View task". I think the standard layout uses a simple button "View task" button, and the light notification layout uses a ``Your task - <link>View [FIX] proutichou</link>``.

This could probably be cleaned a bit furthermore, but it does the job currently.